src/main/java/org/mdbenefits/app/data/enums/HelpNeededType.java

```diff
@@ -3,6 +3,7 @@
 import static org.mdbenefits.app.data.enums.ProgramType.SNAP;
 import static org.mdbenefits.app.data.enums.ProgramType.TCA;
 import static org.mdbenefits.app.data.enums.ProgramType.TDAP;
+import static org.mdbenefits.app.data.enums.ProgramType.RCA;
 import static org.mdbenefits.app.data.enums.ProgramType.OTHER;
```

Comment:
To do: make "other option" a "none of the above"

Reply:
Since "other" will act as none of the above, it might make sense to remove from the HelpNeededType.

I can add a separate checkBoxInSet that is noneOfTheAbove, and therefore saves as an empty list. 

On PreCheckPrograms, we can skip the logic if HelpNeeded is empty.

Then in the chosePrograms template, I can add a different logic that uses all of the HelpNeededType when program is empty.

OR

There can be another screen chooseProgramsOther that always displays everything on the HelpNeededType.